### PR TITLE
Use common challenge

### PR DESCRIPTION
**Pull request by zauguin**
Switch back to the U2F API v1.1 `sign` interface to fix Chrome authentification.(#45)
Additionally patch `u2flib_server` to be more compatible with v1.1.

### CODE REVIEW

File: vendor/yubico/u2flib-server/src/u2flib_server/U2F.php [hunk 1]
```diff
@@ -222,6 +222,7 @@ public function doRegister($request, $response, $includeCert = true)
     public function getAuthenticateData(array $registrations)
     {
         $sigs = array();
+        $challenge = $this->createChallenge();
```
Comment (ChristophWurst) [inline vendor/yubico/u2flib-server/src/u2flib_server/U2F.php]: Makes sense!

However, we shoud try to get this merged in the upstream repo https://github.com/Yubico/php-u2flib-server ;-)

Comment (zauguin) [inline vendor/yubico/u2flib-server/src/u2flib_server/U2F.php]: Proposed as Yubico/php-u2flib-server#55

**Comment by Nibbels** [comment]
If you make a new release I will test this instantly.
However my main PC never had any problem with two sticks. But I might be able to grab another laptop too. 	Nevertheless both Systems have Win 10.

**Comment by ChristophWurst** [comment]
I can confirm that this lets me log into Nextcloud on both FF and Chrome with all my registered keys. Thanks a lot!

**Comment by ChristophWurst** [comment]
I have updated the u2f lib in #49. Let's continue there 😉

Not that you created the PR from a branch of your personal fork. This means I cannot push to it. You do have permissions to push a branch directly to this repo though, as you're one of the collaborators!

**Pull request closed without merging**